components/dashboard/src/components/PillLabel.tsx

```diff
@@ -4,21 +4,27 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
+export type PillType = "info" | "warn" | "success";
```

Comment:
**thought**: Seeing the expand button to read the whole file is probably a good sign we may have to shrink this file by reusing components, etc. 💭

Reply:
@gtsiolis not sure what you mean, is this the correct file you are referring to?

Reply:
@jeanp413 Wrong file, indeed! This comment was about `PersonalAccessTokens.tsx` file. However, feel free to leave this as is for this iteration. By expand, I mean the _Load diff_ button as seen below.

<img width="770" alt="Screenshot 2022-11-30 at 9 36 36 PM" src="https://user-images.githubusercontent.com/120486/204892953-8cef6431-7c8d-4b93-a9b6-1b0b1d74b678.png">